### PR TITLE
Internal: Hide 'Getting Started' menu item if Home is active [ED-14668]

### PR DESCRIPTION
**Pull request by TzviRabinovitch**

### CODE REVIEW

File: tests/playwright/pages/wp-admin-page.ts [hunk 1]
```diff
@@ -178,12 +178,18 @@ export default class WpAdminPage extends BasePage {
 	 *
 	 * TODO: The testing environment isn't clean between tests - Use with caution!
 	 *
-	 * @param {Object} experiments - Experiments settings ( `{ experiment_id: true / false }` );
+	 * @param {Object}            experiments - Experiments settings ( `{ experiment_id: true / false }` );
```
Comment (davseve) [inline tests/playwright/pages/wp-admin-page.ts:181]: Is it in purpose?

Comment (TzviRabinovitch) [reply tests/playwright/pages/wp-admin-page.ts:181]: lint throw an error

**Review by davseve — DISMISSED** [review]

File: tests/playwright/pages/wp-admin-page.ts [hunk 2]
```diff
@@ -178,12 +178,18 @@ export default class WpAdminPage extends BasePage {
...
+	 * @param {(boolean|string)=} oldUrl
```
Comment (rami-elementor) [inline tests/playwright/pages/wp-admin-page.ts:182]:
```suggestion
	 * @param {(boolean|string)=} oldUrl - Optional. Whether to use the old URL structure. Default is false.
```

File: includes/settings/admin-menu-items/getting-started-menu-item.php [hunk 3]
```diff
@@ -12,6 +12,9 @@
 
 class Getting_Started_Menu_Item implements Admin_Menu_Item_With_Page {
 	public function is_visible() {
+		if ( Plugin::instance()->experiments->is_feature_active( 'home_screen' ) ) {
+			return false;
+		}
 		return true;
```
Comment (ronenelementor) [inline includes/settings/admin-menu-items/getting-started-menu-item.php:18]:
```suggestion
		if ( Plugin::instance()->experiments->is_feature_active( 'home_screen' ) ) {
			return false;
		}

		return true;
```

File: tests/playwright/pages/wp-admin-page.ts [hunk 4]
```diff
@@ -178,12 +178,18 @@ export default class WpAdminPage extends BasePage {
...
 	 *
-	 * @param {Object} experiments - Experiments settings ( `{ experiment_id: true / false }` );
+	 * @param {Object}            experiments - Experiments settings ( `{ experiment_id: true / false }` );
+	 * @param {(boolean|string)=} oldUrl - Optional. Whether to use the old URL structure. Default is false.
 	 *
 	 * @return {Promise<void>}
 	 */
-	async setExperiments( experiments: {[ n: string ]: boolean | string } ) {
-		await this.page.goto( '/wp-admin/admin.php?page=elementor-settings#tab-experiments' );
+	async setExperiments( experiments: {[ n: string ]: boolean | string }, oldUrl:boolean = false ) {
```
Comment (ronenelementor) [inline tests/playwright/pages/wp-admin-page.ts:186]:
```suggestion
	async setExperiments( experiments: { [ n: string ]: boolean | string }, oldUrl: boolean = false ) {
```

File: tests/playwright/sanity/elemenetor-menu.test.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+async function validateGettingStartedLinkCount( wpAdmin: WpAdminPage, expectedCount ) {
+	await wpAdmin.page.locator( 'li .toplevel_page_elementor' ).click();
+	expect( await wpAdmin.page.getByText( 'Getting Started' ).count() ).toEqual( expectedCount );
+}
+
+test.describe( 'General Settings', () => {
+	test( 'Is hidden if home is active (default for plugin users)', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.gotoDashboard();
```
Comment (ronenelementor) [inline tests/playwright/sanity/elemenetor-menu.test.ts:18]:
```suggestion
		const wpAdmin = new WpAdminPage( page, testInfo );

		await wpAdmin.gotoDashboard();
```

File: tests/playwright/sanity/elemenetor-menu.test.ts [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+		await validateGettingStartedLinkCount( wpAdmin, 0 );
+		await validateGettingsStartedPage( wpAdmin );
+	} );
+
+	test( 'Is visible if home is not active (default for hosting users)', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( { home_screen: false } );
```
Comment (ronenelementor) [inline tests/playwright/sanity/elemenetor-menu.test.ts:26]:
```suggestion
		const wpAdmin = new WpAdminPage( page, testInfo );

		await wpAdmin.setExperiments( { home_screen: false } );
```

File: includes/settings/admin-menu-items/getting-started-menu-item.php (hunk 3 above)
Comment (ronenelementor) [inline includes/settings/admin-menu-items/getting-started-menu-item.php:18]:
```suggestion
		return ! Plugin::instance()->experiments->is_feature_active( 'home_screen' );
```

**Review by ronenboxer — APPROVED** [review]

**Pull request merged**